bigquery_storage/to_dataframe/main_test.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import uuid
+
+import pytest
+
+
+@pytest.fixture
+def clients():
+    # [START bigquerystorage_pandas_tutorial_create_client]
+    import google.auth
+    from google.cloud import bigquery
+    from google.cloud import bigquery_storage_v1beta1
+
+    # Explicitly create a credentials object. This allows you to use the same
+    # credentials for both the BigQuery and BigQuery Storage clients, avoiding
+    # unnecessary API calls to fetch duplicate authentication tokens.
+    credentials, project_id = google.auth.default(
+        scopes=["https://www.googleapis.com/auth/cloud-platform"]
+    )
+
+    # Make clients.
+    bqclient = bigquery.Client(credentials=credentials, project=project_id)
+    bqstorageclient = bigquery_storage_v1beta1.BigQueryStorageClient(
+        credentials=credentials
+    )
+    # [END bigquerystorage_pandas_tutorial_create_client]
+    return bqclient, bqstorageclient, project_id
+
+
+@pytest.fixture
+def temporary_dataset(clients):
+    bqclient, _, _ = clients
+    dataset_id = "bqstorage_to_dataset_{}".format(uuid.uuid4().hex)
+
+    # [START bigquerystorage_pandas_tutorial_create_dataset]
+    from google.cloud import bigquery
+
+    # TODO: Set the dataset_id to the dataset used to store temporary results.
+    # dataset_id = "query_results_dataset"
+
+    dataset_ref = bqclient.dataset(dataset_id)
+    dataset = bigquery.Dataset(dataset_ref)
+
+    # Remove tables after 24 hours.
+    dataset.default_table_expiration_ms = 1000 * 60 * 60 * 24
+
+    bqclient.create_dataset(dataset)  # API request.
+    # [END bigquerystorage_pandas_tutorial_create_dataset]
+    yield dataset_id
+    bqclient.delete_dataset(dataset_ref, delete_contents=True)
+
+
+def test_table_to_dataframe(capsys, clients):
+    bqclient, bqstorageclient, _ = clients
+
+    # [START bigquerystorage_pandas_tutorial_read_table]
+    from google.cloud import bigquery
```

Comment:
I thought since it's a tutorial you were gonna just have these imports in the client initialization snippet?